index.js

```diff
@@ -3,5 +3,5 @@
 module.exports = {
   src: require('./lib/src'),
   dest: require('./lib/dest'),
-  symlink: require('./lib/symlink')
+  symlink: require('./lib/symlink'),
```

Comment:
https://github.com/gulpjs/gulp/blob/master/bin/gulp.js#L25

I think it's silly, but your other repo's use it. Can't remember if it was via ESLint or jacks though.